content/en/community/mission.md

```diff
@@ -1,6 +1,7 @@
 ---
 title: Mission, vision, and values
```

Comment:
```suggestion
title: Mission, vision, and values
linkTitle: Mission
```

Reply:
Or maybe you'd prefer a `linkTitle` like "Mission ..." to show that there's more to that page title?

Reply:
As much as I agree with you that this title is to long, I would leave it like that, since it's an "official" title and I remember that we had this discussion before. @cartermp @austinlparker, wdyt?